src/abConfiguration.js

```diff
@@ -1,66 +1,66 @@
-var ABConfiguration = (function() { // jshint ignore:line
```

Comment:
most of the changes to the code is formatting, you should review with whitespace off.